src/CommandCollector.php

```diff
@@ -15,7 +15,7 @@ class CommandCollector
     public function processCommandServices(array $services)
     {
         foreach ($services as $service) {
-            $this->commands[] = CommandReflection::fromClass($service);
+            $this->commands[] = CommandReflection::fromClass($service['handles']);
```

Comment:
Create new service for resolving from class name to form type. Feeded by compiler pass.